### PR TITLE
fix(rutorrent): Ensure /srv is created, get rid of cds, remove cloudscraper and python dep

### LINKED ISSUE #669: rutorrent url 404
## Bug description
Receiving upon IP/rutorrent

Not Found
The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.

## Reproduction steps
Installed nginx first
Installed panel
Installed rtorrent and rutorrent
Ran box update

## Installed applications through swizzin
nginx
panel
rtorrent
rutorrent

## Expected behavior
Clicking on rutorrent via panel to load the interface

## Server Info
Ubuntu 18.04 64bit
Using swizzin from github

## Is Swizzin up to date?
Commit hash: abff162

## Additional context
rtorrent.conf not in /etc/nginx/apps/ until I run box update

### PR DESCRIPTION
**Pull request by flying-sausages**
## Fixes issues: 
- `cd` failing when `/srv` is not present (so Ubuntu)
- Cloudscraper removed
- rutorrent updater taking long af
- rut installer not failing gracefully

## Proposed Changes:
- `mkdir -p /srv`
- Stop using `cd` where possible
- more failure catches
- Check install locks

## Change Categories

- Bug fix 

## Checklist

- [x] Docs have been made OR are not necessary
    - PR link: 
- [x] Changes to panel have been made OR are not necessary
    - PR link: 
- [x] Code is formatted [(See more)](https://github.com/swizzin/swizzin/blob/master/CONTRIBUTING.md#editor-plugins-and-tooling)
- [x] Code conforms to project structure [(See more)](https://swizzin.ltd/dev/structure)
- [x] Shellcheck isn't screaming [(See more)](https://github.com/swizzin/swizzin/blob/master/CONTRIBUTING.md#editor-plugins-and-tooling)
- [x] Prints to terminal are handled [(See more)](https://github.com/swizzin/swizzin/blob/master/CONTRIBUTING.md#printing-into-the-terminal)
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have functionally tested the changes being introduced

### Architectures

|   			| `amd64` 	| `armhf` 	| `arm64` 	| Unspecified 	|
|--------		|-------- 	|-------- 	|-------- 	|----------		|
| Focal 		|	✅ 		|			|			|				|
| Bionic		|			|			|			|				|
| Buster		|			|			|			|				|
| Stretch		|			|			|			|				|
| Raspbian  	|	⚫️		|			|	⚫️		|	⚫️			|

### ✅❎ Passed
- Fresh install on focal
image

### CODE REVIEW

**Comment by flying-sausages** [comment]
image
Now even faster without cloudscraper or the python dep

File: sources/functions/utils [hunk 1]
```diff
@@ -6,10 +6,14 @@ function install_rar() {
     if ! which unrar > /dev/null; then
         case "$(_os_arch)" in
             "amd64")
-                apt_install rar unrar
+                apt_install rar unrar || {
+                    exit 1
```
Comment (liaralabs) [inline sources/functions/utils:10]: This is superfluous, no? Doesn't apt already exist the current installer on error?

Comment (flying-sausages) [reply sources/functions/utils:10]: Oh yeah, should be

Comment (flying-sausages) [inline sources/functions/utils:10]:
```suggestion
                apt_install rar unrar
```

**Review by liaralabs — APPROVED** [review]
I altered when the Club QuickBox echo is displayed. Otherwise, looks good to me. Tests appear fine and `box update` as well. If you're okay with the Club QB changes then merge away.

**Pull request merged**